tests/input_files/contact3D_exodus_in.4C.yaml

```diff
@@ -0,0 +1,82 @@
+PROBLEM SIZE:
+  ELEMENTS: 2000
+  NODES: 2662
+  MATERIALS: 2
+  NUMDF: 6
+PROBLEM TYPE:
+  PROBLEMTYPE: "Structure"
+
+STRUCTURE GEOMETRY:
+  FILE: "contact3D_exodus_in.exo"
+  ELEMENT_BLOCKS:
+    - ID: 1
+      ELEMENT_NAME: SOLID
+      ELEMENT_DATA: "MAT 1 KINEM nonlinear"
+    - ID: 2
+      ELEMENT_NAME: SOLID
+      ELEMENT_DATA: "MAT 2 KINEM nonlinear"
+
+STRUCTURAL DYNAMIC:
+  INT_STRATEGY: "Standard"
+  TIMESTEP: 0.02
+  NUMSTEP: 6
+  MAXTIME: 1
+  DAMPING: "Rayleigh"
+  M_DAMP: 1e-05
+  K_DAMP: 1e-05
+  TOLDISP: 1e-08
+  LINEAR_SOLVER: 1
+  # Important setting: ensure that we use the file input
+  GEOMETRY: file
+CONTACT DYNAMIC:
+  LINEAR_SOLVER: 2
+SOLVER 1:
+  SOLVER: "UMFPACK"
+  NAME: "Structure_Solver"
+SOLVER 2:
+  SOLVER: "UMFPACK"
+  NAME: "Contact_Solver"
+MATERIALS:
+  - MAT: 1
+    MAT_Struct_StVenantKirchhoff:
+      YOUNG: 100
+      NUE: 0.3
+      DENS: 0.5
+  - MAT: 2
+    MAT_Struct_StVenantKirchhoff:
+      YOUNG: 4000
+      NUE: 0.3
+      DENS: 1
+FUNCT1:
+  - COMPONENT: 0
+    SYMBOLIC_FUNCTION_OF_SPACE_TIME: "a"
+  - VARIABLE: 0
+    NAME: "a"
+    TYPE: "linearinterpolation"
+    NUMPOINTS: 3
+    TIMES: [0, 0.1, 2]
+    VALUES: [0, 0.02, 0.02]
+
+DESIGN SURF MORTAR CONTACT CONDITIONS 3D:
+  - E: 1
+    ENTITY_TYPE: node_set_id
+    InterfaceID: 1
+    Side: Slave
+  - E: 2
+    ENTITY_TYPE: node_set_id
+    InterfaceID: 1
+    Side: Master
```

Comment:
3. Boundary conditions gained the new parameter `ENTITY_TYPE` which tells us, what the number `E` actually means. By default, if you specify nothing, the `ENTITIY_TYPE` is `legacy_id` which refers to the `DVOL-NODE` etc. topology which was hand-written. If you specify `node_set_id`, as was done here, we refer to a node set in the exodus file. This is much simpler and does not require the indirect renumbering we did before.